docs/paas-plus/dev/integrate-spryker-code-upgrader.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Integrate Spryker Code Upgrader into your development process
```

Comment:
Same

Reply:
Does it mean that the filename should be exact the same as the title, but only with - instead of spaces?